invenio_jsonschemas/ext.py

```diff
@@ -165,6 +166,14 @@ def path_to_url(self, path):
         ).build(
             'schema', values={'path': path}, force_external=True)
 
+    @cached_property
+    def loader(self):
+        """Loader used by `JsonRef.replace_refs`."""
+        loader_cls = self.app.config['JSONSCHEMAS_LOADER']
+        if isinstance(loader_cls, six.string_types):
+            return import_string(loader_cls)
```

Comment:
The only thing missing would be:

```diff
diff --git a/invenio_jsonschemas/ext.py b/invenio_jsonschemas/ext.py
index 60e5950..6e8c797 100644
--- a/invenio_jsonschemas/ext.py
+++ b/invenio_jsonschemas/ext.py
@@ -29,6 +29,7 @@ from __future__ import absolute_import, print_function
 import json
 import os

+import jsonref
 import pkg_resources
 import six
 from six.moves.urllib.parse import urlsplit
@@ -172,7 +173,7 @@ class InvenioJSONSchemasState(object):
         cls = self.app.config['JSONSCHEMAS_LOADER_CLS']
         if isinstance(cls, six.string_types):
             return import_string(cls)
-        return cls
+        return jsonref.JsonLoader
```

Otherwise when `JSONSCHEMAS_REPLACE_REFS` is set to True and `JSONSCHEMAS_LOADER_CLS` is None an error will be raised when trying to create an instance of `None`